src/docs/concepts/gauges.adoc

```diff
@@ -35,13 +35,13 @@ myGauge.set(27);
 myGauge.set(11);
 ----
 
-Note that in this form unlike other meter types you don't get a reference to the `Gauge` when creating one, but rather the thing being observed. This is because of the heisen-gauge principal; the gauge is self sufficient once created, so you should never need to interact with it. This allows us to give you back only the instrumented object, which allows for quick one liners that both create the object to be observed and set up metrics around it.
+Note that, in this form, unlike other meter types, you do not get a reference to the `Gauge` when creating one. Rather, you get a reference to the thing being observed. This is because of the "`heisen-gauge`" principal: The gauge is self-sufficient once created, so you should never need to interact with it. This lets us give you back only the instrumented object, which allows for quick one-liners that both create the object to be observed and set up metrics around it.
```

Comment:
The combination of back-ticks and quotation marks produces curly quotation marks when it's rendered. It may look odd in an editor (it ends up being italicized in my editor, for example), but it's the final rendering that matters.